packages/blazor-workspace/NimbleBlazor/CONTRIBUTING.md

```diff
@@ -10,18 +10,18 @@ Initialize and build the Nimble monorepo (`npm install` + `npm run build` from t
 
 For Nimble Blazor development on Windows, the suggested tools to install are:
 - Visual Studio 2022 ([Enterprise, if available](https://my.visualstudio.com/Downloads?PId=8229)): Choose the "ASP.NET and Web Development" Workload in the installer
-- Ensure Visual Studio is completely up to date (v17.1.6+): In Visual Studio click "Help" then "Check for Updates"
+- Ensure Visual Studio is completely up to date (v17.11.2+): In Visual Studio click "Help" then "Check for Updates"
 - (Optional) Enable IIS (see "Enabling IIS", below)
-- ASP.NET Core Runtime 6.0.4xx: Choose "Hosting Bundle" under ASP.NET Core Runtime, on the [.NET 6.0 Download Page](https://dotnet.microsoft.com/en-us/download/dotnet/6.0)
+- ASP.NET Core Runtime 8.0.x: Choose "Hosting Bundle" under ASP.NET Core Runtime, on the [.NET 8.0 Download Page](https://dotnet.microsoft.com/en-us/download/dotnet/8.0)
 
-In Visual Studio, run either the `Demo.Server` or `Demo.Client` project to see the Blazor demo apps.
+In Visual Studio, run the `Demo.Server`, `Demo.Client`, or `Demo.Hybrid` projects to see the Blazor demo apps.
```

Comment:
I can run Server and Hybrid in VS 2022 just fine, but if I set client as default and run I get the following:
![image](https://github.com/user-attachments/assets/35f9f48e-92ce-4893-93e0-9d45ea57c1a9)

output window:
```
Microsoft.Hosting.Lifetime: Information: Now listening on: https://localhost:7026
Microsoft.Hosting.Lifetime: Information: Now listening on: http://localhost:5026
Microsoft.Hosting.Lifetime: Information: Application started. Press Ctrl+C to shut down.
Microsoft.Hosting.Lifetime: Information: Hosting environment: Development
Microsoft.Hosting.Lifetime: Information: Content root path: C:\dev\github\nimble\packages\blazor-workspace\Examples\Demo.Client
Verbose logs are written to:
C:\Users\mraj\AppData\Local\Temp\visualstudio-js-debugger.txt
The program '' has exited with code 4294967295 (0xffffffff).
The program '[55672] dotnet.exe: Program Trace' has exited with code 0 (0x0).
The program '[55672] dotnet.exe' has exited with code 4294967295 (0xffffffff).
```

Reply:
Looks like the above happens in Demo.Client with Debug and not with "Start without Debugging"

Reply:
Yeah I've seen the same. Think I should add that to the docs? (Haven't looked to see if it's a known issue / bug with Blazor yet)

Reply:
Added a note about this situation to the docs.